core/src/main/scala/kafka/common/ZkNodeChangeNotificationListener.scala

```diff
@@ -92,7 +92,7 @@ class ZkNodeChangeNotificationListener(private val zkUtils: ZkUtils,
           if (changeId > lastExecutedChange) {
             val changeZnode = seqNodeRoot + "/" + notification
             val (data, stat) = zkUtils.readDataMaybeNull(changeZnode)
-            data map (notificationHandler.processNotification(_)) getOrElse (logger.warn(s"read null data from $changeZnode when processing notification $notification"))
+            data map (notificationHandler.processNotification(_)) getOrElse logger.warn(s"read null data from $changeZnode when processing notification $notification")
```

Comment:
I think I would leave this as it was or remove infix syntax as we don't normally use it, e.g.:

```scala
data.map(notificationHandler.processNotification(_)).getOrElse {
  logger.warn(s"read null data from $changeZnode when processing notification $notification")
}
```

I also moved the logger to the following line as the line is too long currently.